app/policies/application_policy.rb

```diff
@@ -17,7 +17,7 @@ def show?
   end
 
   def create?
-    false
+    true
```

Comment:
I think we're not supposed to change the application_policy file, only the kondo_policy

Reply:
I'll leave that to you 🤣 I needed temporarily change that to test if create kondo works 😁

Reply:
Lol it's strange, I thought the pundit would stop once it finds the create method in kondo_policy... Then it means application_policy overwrite everything? haha I am confused now

def create?
  current_user.provider?
end

Reply:
I was actually wondering that myself, and then again I didnt want to spend a lot of time to debug it so I just temporarily set it to true haha

Reply:
So weird 😂😂 anyway I’ll merge your branch, we can always set it back to false later!